compass/landice/tests/greenland/mesh.py

```diff
@@ -80,3 +129,22 @@ def run(self):
                                 'southGreenland',
                                 'southWestGreenland',
                                 'westCentralGreenland'])
+
+        # is there a way to encompass this in an existing framework function?
+        data = netCDF4.Dataset('GIS.nc', 'r+')
+        data.set_auto_mask(False)
+        # Ensure basalHeatFlux is positive
+        data.variables['basalHeatFlux'][:] = np.abs(
+            data.variables['basalHeatFlux'][:])
+        # Ensure reasonable dHdt values
+        dHdt = data.variables["observedThicknessTendency"][:]
+        dHdtErr = data.variables["observedThicknessTendencyUncertainty"][:]
+        # Arbitrary 5% uncertainty; improve this later
+        dHdtErr = np.abs(dHdt) * 0.05
+        # large uncertainty where data is missing
+        dHdtErr[np.abs(dHdt) > 1.0] = 1.0
+        dHdt[np.abs(dHdt) > 1.0] = 0.0  # Remove ridiculous values
+        data.variables["observedThicknessTendency"][:] = dHdt
+        data.variables["observedThicknessTendencyUncertainty"][:] = dHdtErr
+
+        data.close()
```

Comment:
I'm having a hard time finding the analogous manipulation to Antarctica mesh within the `antarctica` testgroup and/or framework functions. 

If this is Greenland specific then I'll wrap with is a function (within the step) and call it a day, but I have a feeling that there's something I'm missing and a similar operation gets done for Antarctica.

Reply:
I think the treatment of `basalHeatFlux` can just be removed, because we've updated the source datasets and added checks on the sign in the interpolation script.
The treatment of `dHdt` and `dHdtErr` are Greenland specific because the gridded dataset for Greenland has missing uncertainties, etc, so don't worry about doing this for Antarctica.